package.json

```diff
@@ -29,7 +29,7 @@
 		{
 			"gzip": false,
 			"running": false,
-			"limit": "7 KB",
+			"limit": "7.083 KB",
```

Comment:
Instead of increasing the limit, is there a way to have the JS logic inlined just for this case, or maybe do some small refactoring so that we keep it under the size limit?

I see a few strings that can be moved into a constants array so that when minified they are being reused instead of duplicated.